RecoLocalCalo/EcalRecProducers/python/ecalUncalibRecHitPhase2_cff.py

```diff
@@ -1,55 +1,50 @@
 import FWCore.ParameterSet.Config as cms
 from HeterogeneousCore.CUDACore.SwitchProducerCUDA import SwitchProducerCUDA
-from Configuration.ProcessModifiers.gpu_cff import gpu
-
 
+# ECAL Phase 2 weights running on CPU
 from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2_cfi import ecalUncalibRecHitPhase2 as _ecalUncalibRecHitPhase2
+ecalUncalibRecHitPhase2CPU = _ecalUncalibRecHitPhase2.clone() 
 ecalUncalibRecHitPhase2 = SwitchProducerCUDA(
```

Comment:
Not necessarily for this PR, but when could the `SwitchProducerCUDA` be removed? By quick look removing looks fairly straightforward (_might_ be as simple as
```py
ecalUncalibRecHitPhase2 = _ecalUncalibRecHitPhase2.clone()
```
here and the `alpaka.toModify(ecalUncalibRecHitPhase2,` further below replaced with
```py
alpaka.toReplaceWith(ecalUncalibRecHitPhase2, _ecalUncalibRecHitSoAToLegacy.clone(
```
)

Reply:
[RecoLocalCalo/EcalRecProducers/test/testEcalUncalibRecHitPhase2WeightsProducerGPU.py](https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/EcalRecProducers/test/testEcalUncalibRecHitPhase2WeightsProducerGPU.py#L80-L83) needs to be updated as well.

Reply:
To clarify: it needs to be updated anyway, because after these changes there is no `@cuda` module any more.